### ISSUE TITLE
Not able to achieve good performance on LVIS dataset

### ISSUE THREAD

**Issue by yoloyash** [THU-MIG/yolov10#385]
Hello, great work on improving the v8 family! I was trying to train YOLOv10m on the LVIS Dataset. While the dataset is quite complex and big, I'm barely able to achieve mAP@0:50:0.95 > 10 after training for 75 epochs. I'm unsure if I'm using the wrong LR/batch or train hyperparameters. I'm training 8 x Nvidia L4 (24GB) with 96 workers. Please let me know if these results are expected or if I can do something to make it better. 

> 
> task: detect
> mode: train
> model: yolov10m.pt
> data: datasets/lvis/dataset.yaml
> epochs: 500
> time: null
> patience: 50
> batch: 96
> imgsz: 640
> save: true
> save_period: -1
> val_period: 1
> cache: false
> device: 0,1,2,3,4,5,6,7
> workers: 96
> project: train
> name: 5_lvis
> exist_ok: false
> pretrained: true
> optimizer: auto
> verbose: true
> seed: 0
> deterministic: true
> single_cls: false
> rect: false
> cos_lr: false
> close_mosaic: 10
> resume: false
> amp: true
> fraction: 1.0
> profile: false
> freeze: null
> multi_scale: false
> overlap_mask: true
> mask_ratio: 4
> dropout: 0.0
> val: true
> split: val
> save_json: false
> save_hybrid: false
> conf: null
> iou: 0.7
> max_det: 300
> half: false
> dnn: false
> plots: true
> source: null
> vid_stride: 1
> stream_buffer: false
> visualize: false
> augment: false
> agnostic_nms: false
> classes: null
> retina_masks: false
> embed: null
> show: false
> save_frames: false
> save_txt: false
> save_conf: false
> save_crop: false
> show_labels: true
> show_conf: true
> show_boxes: true
> line_width: null
> format: torchscript
> keras: false
> optimize: false
> int8: false
> dynamic: false
> simplify: false
> opset: null
> workspace: 4
> nms: false
> lr0: 0.01
> lrf: 0.01
> momentum: 0.937
> weight_decay: 0.0005
> warmup_epochs: 3.0
> warmup_momentum: 0.8
> warmup_bias_lr: 0.1
> box: 7.5
> cls: 0.5
> dfl: 1.5
> pose: 12.0
> kobj: 1.0
> label_smoothing: 0.0
> nbs: 64
> hsv_h: 0.015
> hsv_s: 0.7
> hsv_v: 0.4
> degrees: 0.0
> translate: 0.1
> scale: 0.5
> shear: 0.0
> perspective: 0.0
> flipud: 0.0
> fliplr: 0.5
> bgr: 0.0
> mosaic: 1.0
> mixup: 0.0
> copy_paste: 0.0
> auto_augment: randaugment
> erasing: 0.4
> crop_fraction: 1.0
> cfg: null
> tracker: botsort.yaml
> save_dir: train/5_lvis
>
